doc/lib_menu.md

```diff
@@ -0,0 +1,31 @@
+// This file is distributed under the terms of the MIT license, (c) the KSLib team
+
+## lib_menu
+
+Use this library when you want to display a menu on the screen.
+
+### open_menu
+
+args:
+  * title - string, a title to display at the top of the dialog.
+  * list_of_names - list of strings, options from which the user will choose.
+
+returns:
+  * string, an option chosen by user.
+
+description:
+  * Show a menu, return user's choice.
+  * This should be preferred to open_menu_indexed when making a menu
+    because strings as return values are more self-explanetory.
```

Comment:
I can pull it into my own branch then add comments and resubmit the PR but that is the only way i am aware of.
I will do that in a sec (my laptop is being really slow with ksp and fire fox running)